### ISSUE TITLE
Docs: Explain how to find IdempotencyValidationError

### ISSUE THREAD

**Issue by rubenfonseca** [awslabs/aws-lambda-powertools-python#4197]
### What were you searching in the docs?

(from Discord)

When searching on how to validate an idempotency request, there's a note:

> If we try to send the same request but with a different amount, we will raise IdempotencyValidationError.

However, it's not explained where the `IdempotencyValidationError` comes from.

### Is this related to an existing documentation section?

https://docs.powertools.aws.dev/lambda/python/latest/utilities/idempotency/#payload-validation

### How can we improve?

The sentence could be expanded to show it comes from 

```python
from aws_lambda_powertools.utilities.idempotency.exceptions import IdempotencyValidationError
```

Or a simple example could be created

### Acknowledgment

- [X] I understand the final update might be different from my proposed suggestion, or refused.

**Comment by leandrodamascena** [comment]
Closed via #4613

**Comment by ChrisHills463** [comment]
> Closed via #4613

Nice work!
